torch/onnx/__init__.py

```diff
@@ -181,6 +187,8 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
         do_constant_folding (bool, default False): Apply the constant-folding optimization.
             Constant-folding will replace some of the ops that have all constant inputs
             with pre-computed constant nodes.
+            The value of parameter ``export_params`` and ``keep_initializers_as_inputs`` might disable
```

Comment:
This comment could be more clear:
"Since this optimization adjusts model initializers, it will be disabled if export_params = Flase or keep_initializers_as_inputs = True."